src/app/charity/charity.component.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../abstract';
+@use '../../abstract';
 
```

Comment:
I guess this isn't needed?

Nice thing about the new use syntax is we can explicitly see when we reference something from abstract and when we don't.
```suggestion
```